libfwupdplugin/fu-bluez-device.c

```diff
@@ -410,6 +416,59 @@ fu_bluez_device_read_battery_interface(FuBluezDevice *self,
 	return TRUE;
 }
 
+/* see https://www.bluetooth.com/specifications/dis-1-2/ spec */
+static gboolean
+fu_bluez_device_parse_device_information_service(FuBluezDevice *self, GError **error)
+{
+	g_autofree gchar *model_number = NULL;
+	g_autofree gchar *serial_number = NULL;
+	g_autofree gchar *fw_revision = NULL;
+	g_autofree gchar *manufacturer = NULL;
+
+	model_number = fu_bluez_device_read_string(self, DI_MODEL_NUMBER_UUID, NULL);
+	if (model_number) {
+		fu_device_add_instance_str(FU_DEVICE(self), "MODEL", model_number);
+		if (!fu_device_build_instance_id_full(FU_DEVICE(self),
+						      FU_DEVICE_INSTANCE_FLAG_GENERIC |
+							  FU_DEVICE_INSTANCE_FLAG_QUIRKS,
+						      error,
+						      "BLUETOOTH",
+						      "MODEL",
+						      NULL)) {
+			g_prefix_error(error, "failed to register model %s: ", model_number);
+			return FALSE;
+		}
+		manufacturer = fu_bluez_device_read_string(self, DI_MODEL_NUMBER_UUID, NULL);
+		if (manufacturer) {
+			fu_device_add_instance_str(FU_DEVICE(self), "MANUFACTURER", manufacturer);
```

Comment:
`VEN` perhaps?

Reply:
Not sure for 100%.
As far as I see manufacturer != vendor.
See from my example -- the manufacturer for "EarFun@ buds is still Qualcomm (however I'm guessing they might leave the defults since the serial is ABCD...)

Reply:
Ya, fair enough.